src/current/molt/molt-fetch.md

```diff
@@ -369,10 +372,71 @@ To cancel replication, enter `ctrl-c` to issue a `SIGTERM` signal. This returns
 `import-only` instructs MOLT Fetch to load the source data in the specified [cloud storage](#cloud-storage) or [local file server](#local-file-server) into the CockroachDB target.
 
 {% include_cached copy-clipboard.html %}
-~~~ 
+~~~ shell
 --mode import-only
 ~~~
 
+#### Fail back to source database
+
+{{site.data.alerts.callout_info}}
+`failback` mode supports MySQL 8.0 and later.
```

Comment:
Super nit: "mode only supports MySQL 8.0". Although with pending work this may change. So keep an eye out for 5.7 support that we're working on. We'll keep you posted.